inc/cartridgeitem_printermodel.class.php

```diff
@@ -189,7 +189,12 @@ static function showForCartridgeItem(CartridgeItem $item) {
                Html::showMassiveActionCheckBox(__CLASS__, $data["id"]);
                echo "</td>";
             }
-            echo "<td class='center'>".$data['type']."</td>";
+            $opt['is_deleted']                  = 0;
```

Comment:
before filling $opt array, you need to initialize it, ex:
```php
$tab = [];
```

or better declare it entirely:
```php
$opt = [
   'is_deleted' => 0,
   'criteria'   => [
      [
         'field'      => 40, // printer model
         'searchtype' => 'equals',
         'value'      => $data["pmid"],
      ]
   ]
];
```